### ISSUE TITLE
Gradle build exception, could not resolve dependencies

### ISSUE THREAD

**Issue by c50a326** [bisq-network/bisq#1537]
Experiencing this with the v0.7.0 tag as well as the v0.6.7
```
FAILURE: Build failed with an exception.

* What went wrong:
A problem occurred configuring root project 'bisq-desktop'.
> Could not resolve all dependencies for configuration ':compile'.
   > Could not find network.bisq:bisq-core:0.7.0.
     Searched in the following locations:
         https://jcenter.bintray.com/network/bisq/bisq-core/0.7.0/bisq-core-0.7.0.pom
         https://jcenter.bintray.com/network/bisq/bisq-core/0.7.0/bisq-core-0.7.0.jar
         https://jitpack.io/network/bisq/bisq-core/0.7.0/bisq-core-0.7.0.pom
         https://jitpack.io/network/bisq/bisq-core/0.7.0/bisq-core-0.7.0.jar
         https://raw.githubusercontent.com/JesusMcCloud/tor-binary/master/release/network/bisq/bisq-core/0.7.0/bisq-core-0.7.0.pom
         https://raw.githubusercontent.com/JesusMcCloud/tor-binary/master/release/network/bisq/bisq-core/0.7.0/bisq-core-0.7.0.jar
         https://dl.bintray.com/jerady/maven/network/bisq/bisq-core/0.7.0/bisq-core-0.7.0.pom
         https://dl.bintray.com/jerady/maven/network/bisq/bisq-core/0.7.0/bisq-core-0.7.0.jar
     Required by:
         project :
   > Could not find network.bisq:bisq-p2p:0.7.0.
     Searched in the following locations:
         https://jcenter.bintray.com/network/bisq/bisq-p2p/0.7.0/bisq-p2p-0.7.0.pom
         https://jcenter.bintray.com/network/bisq/bisq-p2p/0.7.0/bisq-p2p-0.7.0.jar
         https://jitpack.io/network/bisq/bisq-p2p/0.7.0/bisq-p2p-0.7.0.pom
         https://jitpack.io/network/bisq/bisq-p2p/0.7.0/bisq-p2p-0.7.0.jar
         https://raw.githubusercontent.com/JesusMcCloud/tor-binary/master/release/network/bisq/bisq-p2p/0.7.0/bisq-p2p-0.7.0.pom
         https://raw.githubusercontent.com/JesusMcCloud/tor-binary/master/release/network/bisq/bisq-p2p/0.7.0/bisq-p2p-0.7.0.jar
         https://dl.bintray.com/jerady/maven/network/bisq/bisq-p2p/0.7.0/bisq-p2p-0.7.0.pom
         https://dl.bintray.com/jerady/maven/network/bisq/bisq-p2p/0.7.0/bisq-p2p-0.7.0.jar
     Required by:
         project :
   > Could not find network.bisq:bisq-common:0.7.0.
     Searched in the following locations:
         https://jcenter.bintray.com/network/bisq/bisq-common/0.7.0/bisq-common-0.7.0.pom
         https://jcenter.bintray.com/network/bisq/bisq-common/0.7.0/bisq-common-0.7.0.jar
         https://jitpack.io/network/bisq/bisq-common/0.7.0/bisq-common-0.7.0.pom
         https://jitpack.io/network/bisq/bisq-common/0.7.0/bisq-common-0.7.0.jar
         https://raw.githubusercontent.com/JesusMcCloud/tor-binary/master/release/network/bisq/bisq-common/0.7.0/bisq-common-0.7.0.pom
         https://raw.githubusercontent.com/JesusMcCloud/tor-binary/master/release/network/bisq/bisq-common/0.7.0/bisq-common-0.7.0.jar
         https://dl.bintray.com/jerady/maven/network/bisq/bisq-common/0.7.0/bisq-common-0.7.0.pom
         https://dl.bintray.com/jerady/maven/network/bisq/bisq-common/0.7.0/bisq-common-0.7.0.jar
     Required by:
         project :

* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output. Run with --scan to get full insights.

* Get more help at https://help.gradle.org

Deprecated Gradle features were used in this build, making it incompatible with Gradle 5.0.
See https://docs.gradle.org/4.6/userguide/command_line_interface.html#sec:command_line_warnings

BUILD FAILED in 24s
```

**Comment by ManfredKarrer** [comment]
Can you update with latest version? I checked in a wrong gradle version just after release time. I also retriggered some builds on travis.

**Comment by c50a326** [comment]
Building master works, building v0.7.0 still fails. It'd be nice to feel like I'm building a "release" version so that I don't have to worry about any bugs that might come into master.

Edit: thank you as always btw!

**Comment by ManfredKarrer** [comment]
I assume building v0.7.0 fails because there are no artefacts on jitpack. You can build locally if you want to build from the release branch. 
@cbeams What would you recommend?

**Comment by cbeams** [comment]
I know what the problem is. It has to do with how versions were set and how tags were named during the release and the way JitPack works. I can respond fully tomorrow. We can ultimately solve this by publishing a `v0.7.0-a` tag across all the projects, with dependency declarations that are consistent and resolvable all the way down. It won't work otherwise. Again, I'll get back about this more fully tomorrow.

**Comment by dmp1ce** [comment]
Yep, I'm experiencing the same thing trying to release version 0.7.0 for the AUR.

**Comment by cbeams** [comment]
Will get back about this today, thanks.

**Comment by cbeams** [comment]
Closing as resolved. Everyone should be able to build bisq-desktop@v0.7.0 from source now. I just built successfully from source myself; please reopen this issue if issues persist.

For those that are interested, here are the details of the fix, taken from the `0.7.0` 'alias' tag added to bisq-common. Such tags were added to bisq-common, bisq-p2p and bisq-core, and effectively solve the problem. We'll have to sort out our tag-naming / version-naming / JitPack dependency resolution strategy a little better before the next release.

```git
$ git show 0.7.0
tag 0.7.0
Tagger: Chris Beams <chris@beams.io>
Date:   Mon May 14 22:40:59 2018 +0200

Alias 0.7.0 to v0.7.0

bisq-desktop was built with a dependency declaration on
network.bisq:bisq-core:0.7.0, but because these dependencies are
resolved via JitPack, and because JitPack has no awareness of Gradle
versions per se, and instead has awareness only of objects in Git
history, e.g. commit hashes and tag names, these declarations resulted
in unresolvable dependencies. They worked when building Bisq from source
locally within the context of a Gradle composite build (which *is* aware
of Gradle version numbers), but fail when JitPack is in the mix.

So this 'alias' tag makes it possible to build bisq-desktop@v0.7.0 from
source, outside the context of a Gradle composite build and with JitPack
dependency resolution in the mix.
-----BEGIN PGP SIGNATURE-----

iQJDBAABCgAtFiEEHjAxMAinjHQ/TkGXPSFPj1vF7XMFAlr59GIPHGNocmlzQGJl
YW1zLmlvAAoJED0hT49bxe1zulQQAJexi4Cdrb/nOsu+jzumkUdGEo6fYM7q7fqh
CaYwDeryx8jyFOz5fQIpNTybenZJuKY4oGJ6MqggU0c4qrRhMeKZIHCiLI20d4Yu
HEGSIO9hyQ+19lFlv6Jrt7arypbSaPuFy4XC8Gu8P0+cq9tibv6FeXGJxbpfi0xP
aF6U5cJo1IiC+JXcH0RQxxa7ZKGuIwQ7jX3e6mfsRLX5Ykwb8cXV3GwXIxT07Og1
z0lPfca3KR63Fnho8P5iJACowL0viqGOY2PRrdTZUzZTt6fmujSZFk85hRI+t9Mf
+UWXQdz3zp2zHnR6w+Uk8pRaI37y/KO24yjS3qOPhoWu8iPqv1iHxHTktAnyxiNe
Igx6KiMpHSopJuK5fFJEPLkAXTuYjWNPULy+wsqR8P8CdQzg1S57BLQi3m+66zZF
l11oDQkwpndPyervWXfh8EM2FX5sj2irNybmJwucE4/b/p+DMg4GX1z0Z98H3CPu
o/v1ByJwfiSqsvlwCK4qYt1cdEAYbrn+GzZVHMWjXoqjqxze4e84nJkCLKJIegYO
oVVckEV+uHpfNf63l5//JGjqlMf0xAsMpF/iJq2J3nxe6EPFxUdjXuVBGMTkvmCK
NliLfYHkCDll8G8bPVC3VBeIkzml51bYY8pDlerYZIK3lT3VL1qpoT6GUpF1OqgT
eED2MZG/
=Ca8U
-----END PGP SIGNATURE-----

tag v0.7.0
Tagger: Manfred Karrer <mk@nucleo.io>
Date:   Fri May 11 19:40:27 2018 +0200

v0.7.0
-----BEGIN PGP SIGNATURE-----

iQIzBAABCgAdFiEEBENuulMm6f8TlJP8QBJQlmprLEYFAlr11YsACgkQQBJQlmpr
LEYlXw/+IgGD9tX4l1kBAjgyvbZ62ync/75zncaA3OLeg/Tf2cRJo+7jSU/ljN29
BW0hO1Grlpb3fxB3oO6cCYvdiLKCIIo6NwXyigR+bYKOcifGxsilZ+Whs1wqjdto
GaobLM+YWGu0t8VDGfGvc+dyjdKknig8x3BFh1IN3odQNN5fRRDL0ROn4/F+FhnE
RIqyQ8QDtPbsbiiEfCl/JeWuc67xlyn9XCUZCTlHn8JqvIkeZk/Vl/ApBSt+1vi/
3usrQYAWF3BUsLWoLk452titiPeYHKyKlDeBe26XKmVT+g3xcaX4ti7FV924+kos
rx5MFk9G6VFu8DYDR/BvsbgrABAXHLznsHMTMU6TusxSKSBXWnSLYDw7pffETGGM
mq4EqBQ4ujRBduakq09PPfTfzn+pl10NPEWb0Geh0CxQgKwcYSd9EYmH9hiXe7m5
DbDckmAiJifF7kyJXBuxhn3W9mMIyuSGAvAvjdosI2KQnJvlvwpPp6FKl6ynsa5a
VfmrSQItzElLU+0SAitpkUG58Qt7cu1ynelFtJtyGJcHVi05KfqPoMNZ4jP4VnhE
Mbjv0QfzVoEcEyCwoc4WPX9LwCpxrZ70B+KsRReqmNdm7m7ufilUzoHVkdbxO9fp
h4Jxa+/f8Cr3OI7nqBbu6Fa/p0IWn6ijbIXhzrXfE+yGBdKXWRA=
=yIU3
-----END PGP SIGNATURE-----
```
